src/store/reducers/loadingReducer.ts

```diff
@@ -0,0 +1,25 @@
+import { Loading } from "./__types__/loadingTypes";
+
+const initialState: Loading = {
+    isLoading: false,
+};
+
+const loadingReducer = (state: Loading = initialState, action: any) => {

```

Comment:
I would recommend having a type for the action parameter, `action.type.split("_").reverse()[0];` while we are using it for the rest of the code, having strong type safety may decrease unwanted nasty errors. we can use `AnyAction` if possible

https://redux.js.org/usage/usage-with-typescript#typing-additional-redux-logic

Reply:
Changes done